docs/notebooks/t-test.ipynb

```diff
@@ -39,7 +39,7 @@
    "cell_type": "markdown",
```

Comment:
<p>And here you can stary saying</p><p> </p><p>"The linear model implicit in the t-test can also be specified without an intercept term" </p><p> </p><p>The words don't need to be those, feel free to choose your own wording. But I think it is important to mark one model uses the intercept and the other doesn't.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>